<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoltageDialog</class>
 <widget class="QDialog" name="VoltageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="infoPanel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignRight">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Show Selection</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QwtPlot" name="plot"/>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="plotSelector"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
